### PR TITLE
Created a Github actions file with pytest and coverage

### LINKED ISSUE #1: Deploy Github Actions
In my other repos I'm using Travis CI but would like to get more familiar with Github Actions. 

Acceptance Criteria:
- A working pytest test suite run via github actions
- Integration with Codecov (Optional)

### PR DESCRIPTION
**Pull request by StingRayZA**
Tests on Python 3.7 & 3.8...

Should fix #1

### CODE REVIEW

**Comment by StingRayZA** [comment]
Hope this is all good - found this issue through http://issuehub.io 😄 

(Wanted to give actions a try myself...)

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Python package
+
+on: [push]
```
Comment (shepherdjay) [inline .github/workflows/pytest.yml]: Can we have it run on pull and pushes to the master branch?

**Review by shepherdjay — CHANGES_REQUESTED** [review]

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (StingRayZA) [reply .github/workflows/pytest.yml]: Sure - I've added that 👍

**Review by shepherdjay — APPROVED** [review]

**Comment by shepherdjay** [comment]
Excellent - Thanks for the contribution. Love getting CI/CD going early and look forward to reviewing this later. For now seems to work 👍

**Pull request merged**